Sources/AccountSDKIOSWeb/Simplified Login/View/Fonts/Inter.swift

```diff
@@ -0,0 +1,13 @@
+//
+// Copyright © 2022 Schibsted.
+// Licensed under the terms of the MIT license. See LICENSE in the project root.
+//
+
+import Foundation
+
+enum Inter: String, CaseIterable {
```

Comment:
Let's add a note that it's a open source font:

```suggestion
// Inter is a Open Font, see https://github.com/rsms/inter
enum Inter: String, CaseIterable {
```